test/input.txt

```diff
@@ -96,7 +96,7 @@
   find bet
   # does not match if none have keyword
   find 23912039120
-  # matching should be case-sensitive
+  # matching should be case-insensitive
```

Comment:
Would have been better if the test cases added consist of matching that requires the conversion of lower-upper cases